tools/gradle/commons/integrations/standard-test.gradle

```diff
@@ -0,0 +1,35 @@
+apply from: rootProject.file('tools/gradle/commons/docker.gradle')
+
+task standardTest {
+    ext {
+        configPath = ''
+        catalogPath = ''
+    }
+
+    doFirst {
+        exec {
+            def imageName = "${extractImageName(project.file('Dockerfile'))}:dev"
```

Comment:
I acknowledge that this pattern didn't start with this PR, but it's very opaque in the calling context that we extract what are essentially context-specific input arguments. Can the imageName be passed in instead? I would also say the same for the other gradle helper we have.

Reply:
so passing args in gradle is kinda a whole thing. i'm fine with doing it here since i'm already passing args. i'm not going to mess with the other helper in this PR though.

Reply:
amazing. thank you 🙏